Was compiled against a different Node.js version using NODE_MODULE_VERSION

**Describe the bug**
Was compiled against a different Node.js version using NODE_MODULE_VERSION.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to the `master` branch.
2. Run `npm install` and then `npm run start`

**Expected behavior**
I should see the IDE.

**Environment (please complete the following information):**
- OS: macOS Catalina
- OS version: 10.15.7 (19H2)
- node-pty version: 0.9.0
- node: v12.8.1
- npm: 6.10.2
- IDE Version: "0.1.1"

**Additional context**

Log:
```
> mad-machine@0.1.1 start /Users/matviihodovaniuk/Documents/MadMachineIDE
> concurrently "npm run web:start" "npm run app:start"

[1] 
[1] > mad-machine@0.1.1 app:start /Users/matviihodovaniuk/Documents/MadMachineIDE
[1] > cross-env NODE_ENV=development node ./config/scripts/dev.app.js
[1] 
[0] 
[0] > mad-machine@0.1.1 web:start /Users/matviihodovaniuk/Documents/MadMachineIDE
[0] > cross-env ENV=dev node ./config/scripts/start.js
[0] 
[0] Starting the development server...
[0] 
[1] ⚠ ｢copy-webpack-plugin｣: unable to locate '/Users/matviihodovaniuk/Documents/MadMachineIDE/src/public/build/lib/lib' at '/Users/matviihodovaniuk/Documents/MadMachineIDE/src/public/build/lib/lib'
[1] /Users/matviihodovaniuk/Library/Application Support
[1] 
[1] innerError { Error: Cannot find module '../build/Debug/pty.node'
[1]     at Module._resolveFilename (internal/modules/cjs/loader.js:584:15)
[1]     at Function.Module._resolveFilename (/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/electron-prebuilt-compile/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/common/reset-search-paths.js:43:12)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:510:25)
[1]     at Module.require (internal/modules/cjs/loader.js:640:17)
[1]     at require (internal/modules/cjs/helpers.js:20:18)
[1]     at Object.<anonymous> (/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/node-pty/lib/unixTerminal.js:30:15)
[1]     at Object.<anonymous> (/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/node-pty/lib/unixTerminal.js:277:3)
[1]     at Module._compile (internal/modules/cjs/loader.js:693:30)
[1]     at Object.require.extensions.(anonymous function) [as .js] (/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/electron-compile/lib/require-hook.js:77:14)
[1]     at Module.load (internal/modules/cjs/loader.js:602:32) code: 'MODULE_NOT_FOUND' }
[1] 
[1] App threw an error during load
[1] Error: The module '/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/node-pty/build/Release/pty.node'
[1] was compiled against a different Node.js version using
[1] NODE_MODULE_VERSION 72. This version of Node.js requires
[1] NODE_MODULE_VERSION 64. Please try re-compiling or re-installing
[1] the module (for instance, using `npm rebuild` or `npm install`).
[1]     at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:160:31)
[1]     at Object.Module._extensions..node (internal/modules/cjs/loader.js:722:18)
[1]     at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:160:31)
[1]     at Module.load (internal/modules/cjs/loader.js:602:32)
[1]     at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:533:3)
[1]     at Module.require (internal/modules/cjs/loader.js:640:17)
[1]     at require (internal/modules/cjs/helpers.js:20:18)
[1]     at Object.<anonymous> (/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/node-pty/lib/unixTerminal.js:26:11)
[1]     at Object.<anonymous> (/Users/matviihodovaniuk/Documents/MadMachineIDE/node_modules/node-pty/lib/unixTerminal.js:277:3)
[1] 
```

--- a/README.md
+++ b/README.md
@@ -35,6 +35,12 @@ To run the app on the local machine install dependencies:
 npm install
 ```
 
+Rebuild native dependencies. [How to install native modules](https://www.electronjs.org/docs/tutorial/using-native-node-modules)
+
+```sh
+./node_modules/.bin/electron-rebuild
+```
+
 Run the app:
 
 ```sh

--- a/package.json
+++ b/package.json
@@ -113,6 +113,7 @@
   "dependencies": {
     "chokidar": "^2.1.5",
     "drivelist": "^8.0.10",
+    "electron": "4.0.0",
     "electron-compile": "^6.4.4",
     "electron-progressbar": "^1.2.0",
     "electron-squirrel-startup": "^1.0.0",

